graphdatascience/session/aura_api_responses.py

```diff
@@ -11,23 +12,25 @@ class SessionDetails:
     id: str
     name: str
     instance_id: str
-    memory: str  # TODO parse session size?
+    memory: str
     status: str
-    host: Optional[str]  # TODO non-optional once dns based routing
-    expiry_date: Optional[str]  # TODO parse time into datetime using dateutil?
-    created_at: str  # TODO parse time into datetime using dateutil?
+    host: Optional[str]
+    expiry_date: Optional[datetime]
+    created_at: datetime
```

Comment:
this is also nice. small cost, better types 👍

Reply:
in theory, this could help us catch bugs if we mess up the datetime fields in Aura, too :)